core/examples/op2.rs

```diff
@@ -0,0 +1,68 @@
+use anyhow::Context;
+use deno_core::anyhow::Error;
+use deno_core::extension;
+use deno_core::op2;
+use deno_core::resolve_path;
+use deno_core::v8;
+use deno_core::FsModuleLoader;
+use deno_core::JsRuntime;
+use deno_core::OpState;
+use std::rc::Rc;
+
+#[op2]
+fn op_use_state(
+  state: &mut OpState,
+  #[global] callback: v8::Global<v8::Function>,
+) -> Result<(), Error> {
+  state.put(callback);
+  Ok(())
+}
+
+extension!(
+  op2_sample,
+  ops = [op_use_state],
+  esm_entry_point = "ext:op2_sample/op2.js",
+  esm = [ dir "examples", "op2.js" ],
+  docs = "A small example demonstrating op2 usage", "Contains one op"
```

Comment:
Make these sentences to read better in generated docs.

```suggestion
  docs = "A small example demonstrating op2 usage.", "Contains one op."
```